pyproject.toml

```diff
@@ -21,17 +21,18 @@ dependencies = [
     "grpcio-health-checking>=1.35.0,<2.0",
     "grpcio>=1.35.0,<2.0",
     "ijson>=3.1.4,<3.3.0",
+    "import-tracker>=3.1.5,<4",
     "munch>=2.5.0,<4.0",
-    "protobuf>=3.19.0,<5",
```

Comment:
The changes here are because I sorted the list which I think will help avoid conflicts in the future

Reply:
Good call. I picked that one somewhat at random. It looks like `1.20` is the lowest I can reasonably install, so I'll pin it there